mixxxdj/website
# Add support for fetching package metadata

Testing instructions:
1. Download https://gist.github.com/cd8e39e13cdccd3e8f7d6c3556568ed8 and put in into some directory (e.g. `/tmp/foo`).
2. `cd` into that directory and run `python -m http.server 7999`
3. Apply this diff and rebuild locally:

```diff
diff --git a/content/pages/download.md b/content/pages/download.md
index 495d906..0013066 100644
--- a/content/pages/download.md
+++ b/content/pages/download.md
@@ -6,6 +6,7 @@ versions:
     name: 2.3.0 Beta
     next_stable_version: 2.3.0
     release_announcement: /news/2020-06-07-mixxx-2-3-beta-released/
+    download_manifest: http://localhost:7999/manifest.json
     downloads:
       - slug: windows
         os: Windows 7 or later
```

This is how it should look:

![Screenshot from 2021-03-26 15-04-50](https://user-images.githubusercontent.com/1834516/112643707-ef618c80-8e44-11eb-8e2a-24431766b1da.png)
